Felienne/hedy
# Teacher emails & styling of emails

- Add styling to all emails.
- Add teachers email when a user is made teacher.
- Closes #738 

**How to test**:
- See email being sent to user made into teacher
- See all other transactional emails having style.

Suggestions on styling are welcome! The stylings have to be done inline in the elements because HTML email is extremely limiting in the amount of CSS it allows and how it allows it.
